src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -164,11 +164,24 @@ private static bool ApplyChangesFromTC33(JObject sarifLog)
 
                     // https://github.com/oasis-tcs/sarif-spec/issues/302
                     MoveAllStackFrameAddressesToLocation(run);
+
+                    // https://github.com/oasis-tcs/sarif-spec/issues/352
+                    RenameMessageIdToId(run);
```

Comment:
> RenameMessageIdToId(run); [](start = 19, length = 26)

is this still part of 04-03? that makes me a bit nervous since 04-03 is a public release, we should consider adding a new future switch case?